source/index.txt

```diff
@@ -6,6 +6,9 @@
 Start Developing with MongoDB
 =============================
 
+.. meta:: 
+   :keywords: r, elixir, prisma, mongoose
```

Comment:
Q: Should we include all languages in this list?

Reply:
That wasn't part of the approved tagging list for this round, but getting the landing page totally tagged is top of my list for Munich.